net/wireguard/src/opnsense/scripts/Wireguard/wg-service-control.php

```diff
@@ -68,10 +68,14 @@ function wg_start($server, $fhandle, $ifcfgflag = 'up')
     }
     mwexecf('/usr/bin/wg setconf %s %s', [$server->interface, $server->cnfFilename]);
 
-    foreach (explode(',', (string)$server->tunneladdress) as $alias) {
-        $proto = strpos($alias, ':') === false ? "inet" : "inet6";
-        mwexecf('/sbin/ifconfig %s %s %s alias', [$server->interface, $proto, $alias]);
-    }
+    /**
+     * The tunneladdress can be empty, so any string with a length of 0 bytes is removed.
+     */
+    $addresses = array_filter(explode(',', (string)$server->tunneladdress), 'strlen');
```

Comment:
I suppose this can be a one-liner as before in the foreach loop

Reply:
Thanks for the suggestion. I followed your advice. (even though my brain has still trouble with long brackets in brackets xD)

Reply:
you get used to this eventually